Purchases/Public/Package.swift

```diff
@@ -74,18 +74,22 @@ private extension PackageType {
     @objc public let storeProduct: StoreProduct
     @objc public let offeringIdentifier: String
 
+    /// The price of this product using ``priceFormatter``.
     @objc public var localizedPriceString: String {
         return storeProduct.localizedPriceString
     }
 
-    @objc public var localizedIntroductoryPriceString: String {
-        if #available(iOS 11.2, macOS 10.13.2, tvOS 11.2, *) {
-            // todo: uncomment when product discounts are supported in product details
-            // return storeProduct.localizedIntroductoryPriceString
-            return ""
-        } else {
-            return ""
+    /// The price of the `StoreProduct.introductoryPrice` formatted using ``priceFormatter``.
+    /// - Returns: `nil` if there is no `introductoryPrice`.
+    @objc public var localizedIntroductoryPriceString: String? {
```

Comment:
[not needed for this PR] I wonder if this is worth having as a property of StoreProduct and bridging it over here

Reply:
Yeah I went back and forth. I guess it makes sense to have all these computed properties in StoreProduct instead 👍

Reply:
Done.